### PR TITLE
CCD Demo: Override image tags so flux will update full stack with lat…

### PR DESCRIPTION
**Pull request by akdasari**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: k8s/demo/common/ccd/ccd-full.yaml [hunk 1]
```diff
@@ -6,7 +6,19 @@ metadata:
   namespace: ccd
   annotations:
     flux.weave.works/automated: 'true'
-    flux.weave.works/tag.java: 'glob:prod-*'
+    flux.weave.works/tag.ccd-full-admin-web: glob:prod-*
+    flux.weave.works/tag.ccd-full-api-gateway: glob:prod-*
+    flux.weave.works/tag.ccd-full-case-management-web: glob:prod-*
```
Comment (smathangi) [inline k8s/demo/common/ccd/ccd-full.yaml]: bsp has slightly different config without release-name prefix. I'm not sure which one is good practice.

File: k8s/demo/common/ccd/ccd-full.yaml [hunk 2]
```diff
@@ -6,7 +6,19 @@ metadata:
...
+    flux.weave.works/tag.ccd-full-data-store-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-definition-store-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-dm-store: glob:prod-*
+    flux.weave.works/tag.ccd-full-draft-store: glob:prod-*
+    flux.weave.works/tag.ccd-full-em-annotation: glob:prod-*
+    flux.weave.works/tag.ccd-full-s2s: glob:prod-*
+    flux.weave.works/tag.ccd-full-sendletter: glob:prod-*
+    flux.weave.works/tag.ccd-full-user-profile-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-definition-importer: glob:prod-*
```
Comment (smathangi) [inline k8s/demo/common/ccd/ccd-full.yaml]: Do we’ve to specify image param explicitly under each pod / container config in the flux file ?

File: k8s/demo/common/ccd/ccd-full.yaml [hunk 3]
```diff
@@ -6,7 +6,19 @@ metadata:
   namespace: ccd
   annotations:
     flux.weave.works/automated: 'true'
-    flux.weave.works/tag.java: 'glob:prod-*'
+    flux.weave.works/tag.ccd-full-admin-web: glob:prod-*
+    flux.weave.works/tag.ccd-full-api-gateway: glob:prod-*
+    flux.weave.works/tag.ccd-full-case-management-web: glob:prod-*
+    flux.weave.works/tag.ccd-full-data-store-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-definition-store-api: glob:prod-*
+    flux.weave.works/tag.ccd-full-dm-store: glob:prod-*
```
Comment (smathangi) [inline k8s/demo/common/ccd/ccd-full.yaml]: auto update of all dependencies required or just ccd components enough?

File: k8s/demo/common/ccd/ccd-full.yaml (hunk 2 above)
Comment (nathan-clark) [reply k8s/demo/common/ccd/ccd-full.yaml]: whats sendletter?

Comment (akdasari) [reply k8s/demo/common/ccd/ccd-full.yaml]: nathan , that was mistake , fixed now

File: k8s/demo/common/ccd/ccd-full.yaml (hunk 3 above)
Comment (akdasari) [reply k8s/demo/common/ccd/ccd-full.yaml]: better update al components

**Pull request merged**